plugins/org.jkiss.dbeaver.ext.cubrid/plugin.xml

```diff
@@ -0,0 +1,171 @@
+		                                </items>
+           	                        </items>
+           	                    </folder>
+	                        </folder>
+	                        <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridView" label="%tree.tviews.node.name" icon="#folder_view" description="%tree.tviews.node.tip" visibleIf="object.dataSource.metaModel.supportsViews(object.dataSource)">
+	                            <items label="%tree.tview.node.name" path="view" property="views" icon="#view">
+	                                <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridTableColumn" label="%tree.columns.node.name" icon="#columns" description="View columns">
+	                                    <items label="%tree.column.node.name" path="attribute" property="attributes" icon="#column">
+	                                    </items>
+	                                </folder>
+	                            </items>
+	                            <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridView" label="%tree.systemViews.node.name" icon="#folder_view" description="%tree.systemViews.node.tip" visibleIf="object.supportsSystemView() &amp;&amp; object.showSystemTableFolder()">
+		                            <items label="%tree.tview.node.name" path="view" property="systemViews" icon="#view">
+		                                <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridTableColumn" label="%tree.columns.node.name" icon="#columns" description="View columns">
+		                                    <items label="%tree.column.node.name" path="attribute" property="attributes" icon="#column">
+		                                    </items>
+		                                </folder>
+		                            </items>
+		                        </folder>
+	                        </folder>
+	                    </items>
+	                    <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridUser" label="%tree.users.node.name" icon="#sequences" description="%tree.users.node.tip">
+	                        <items label="%tree.user.node.name" path="cubridUser" property="cubridUsers" icon="#user"/>
+	                    </folder>
+                        <folder type="org.jkiss.dbeaver.model.struct.DBSDataType" label="%tree.dataTypes.node.name" icon="#data_types" description="%tree.dataTypes.node.tip">
+                            <items label="%tree.dataType.node.name" path="dataType" property="dataTypes" icon="#data_type"/>
+                        </folder>
+                    </items>
+                </items>
+            </tree>
+            
+            <drivers managable="true">
+                <driver
+                        id="cubrid_jdbc"
+                        label="CUBRID"
+                        icon="icons/cubrid_icon.png"
+                        iconBig="icons/cubrid_icon_big.png"
+                        class="cubrid.jdbc.driver.CUBRIDDriver"

```

Comment:
You can also add logoImage icon here, this will allow logo to be shown in connection settings without the need to override generic connection page

Reply:
I have fixed it and removed Connection page configuration.